### PR TITLE
fix: Kill healthcheck conn after done

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

 Kill healthcheck conn after done

### CODE REVIEW

**Review by delgado3d — APPROVED** [review]

File: lib/realtime/tenants.ex [hunk 1]
```diff
@@ -104,6 +104,7 @@ defmodule Realtime.Tenants do
           end
         end)
 
+        Process.exit(db_conn, :kill)
```
Comment (CarelFdeWaal) [inline lib/realtime/tenants.ex:107]: Not sure if db_conn will be restarted if the process is killed. Perhaps `Genserver.stop(db_conn)` might make sense here(https://stackoverflow.com/questions/44702375/how-to-disconnect-postgrex-connection/44703349#44703349).
To void error first check if pid is alive: https://hexdocs.pm/elixir/1.12/Process.html#alive?/1

That said I did see this pattern in other places in the code, so testing it in iex would be the best

Comment (filipecabaco) [reply lib/realtime/tenants.ex]: actually I will start switching to use GenServer.stop as we been bitten before by Process.exit due to a :normal vs :kill exit (didn't kill child processes)

**Review by CarelFdeWaal — APPROVED** [review]

**Comment by CarelFdeWaal** [comment]
@filipecabaco for interest sake:
```
iex(pink@127.0.0.1)26> {:ok, pid} = Postgrex.start_link(hostname: "localhost", username: "postgres", password: "postgres", database: "postgres")
{:ok, #PID<0.1442.0>}
iex(pink@127.0.0.1)27> Process.info(self())[:links]
[#PID<0.1442.0>]
iex(pink@127.0.0.1)28> GenServer.stop(pid)
:ok
iex(pink@127.0.0.1)29> Process.info(self())[:links]
[]
iex(pink@127.0.0.1)30> {:ok, pid} = Postgrex.start_link(hostname: "localhost", username: "postgres", password: "postgres", database: "postgres")
{:ok, #PID<0.1462.0>}
iex(pink@127.0.0.1)31> Process.exit(pid, :kill)
** (EXIT from #PID<0.1417.0>) shell process exited with reason: killed

Interactive Elixir (1.16.0) - press Ctrl+C to exit (type h() ENTER for help)
```
Because the link exist to the Process.exit also restarts the current iex session .This does not happen with GenServer.stop

**Comment by filipecabaco** [comment]
@CarelFdeWaal that is expected. With that said using GenServer.stop will make things more solid 👍

**Comment by CarelFdeWaal** [comment]
> @filipecabaco for interest sake:
> 
> ```
> iex(pink@127.0.0.1)26> {:ok, pid} = Postgrex.start_link(hostname: "localhost", username: "postgres", password: "postgres", database: "postgres")
> {:ok, #PID<0.1442.0>}
> iex(pink@127.0.0.1)27> Process.info(self())[:links]
> [#PID<0.1442.0>]
> iex(pink@127.0.0.1)28> GenServer.stop(pid)
> :ok
> iex(pink@127.0.0.1)29> Process.info(self())[:links]
> []
> iex(pink@127.0.0.1)30> {:ok, pid} = Postgrex.start_link(hostname: "localhost", username: "postgres", password: "postgres", database: "postgres")
> {:ok, #PID<0.1462.0>}
> iex(pink@127.0.0.1)31> Process.exit(pid, :kill)
> ** (EXIT from #PID<0.1417.0>) shell process exited with reason: killed
> 
> Interactive Elixir (1.16.0) - press Ctrl+C to exit (type h() ENTER for help)
> ```
> 
> Because the link exist to the Process.exit also restarts the current iex session .This does not happen with GenServer.stop

**Comment by CarelFdeWaal** [comment]
Sorry misclick 🙈

**Pull request merged**